### ISSUE TITLE
What does this do? Highlight preservation in RAW settings.

### ISSUE THREAD

**Issue by Thanatomanic** [Beep6581/RawTherapee#4943]
I am a bit lazy here, but this is easier to ask than to look up in the code: which setting is copied when you tick `Highlight preservation`? I can't seem to find a corresponding setting in the editor.

![image](https://user-images.githubusercontent.com/6567747/48023649-b4107700-e13e-11e8-9a28-40f184721547.png)

**Comment by heckflosse** [comment]
@Thanatomanic Good catch. No detailed answer from me atm. Looks wrong. Have to investigate.

**Comment by ff2000** [comment]
Probably Highlight Compression?

**Comment by iliasg** [comment]
It's a remnant of the initial "White point correction" implementation which included a highlight preservation slider .. which were compressing highlights in order to not be clipped.
Sometimes I which it was still there ;)

**Comment by heckflosse** [comment]
I will remove all the related stuff from code now

**Comment by heckflosse** [comment]
For reference:
https://github.com/Beep6581/RawTherapee/issues/2118

**Comment by heckflosse** [comment]
Currently the raw.preser setting is still used in code. Usually it's zero, but there may be very old (more than 3 years old) pp3 files where it is != zero. Should we still support this setting in old pp3 files?

**Comment by Hombre57** [comment]
@heckflosse On my side it depends on how good the effect of this parameter is.

**Comment by heckflosse** [comment]
@Hombre57 The parameter is deprecated since 3 years.

**Comment by Hombre57** [comment]
@heckflosse Then I'll let you decide :wink:
